src/pf-tooltip/pf-tooltip.component.js

```diff
@@ -0,0 +1,304 @@
+import {default as tmpl} from 'pf-tooltip.template';
+import {pfUtil} from 'pf-utils';
+
+/**
+ * <b>&lt;pf-tooltip&gt;</b> element for Patternfly Web Components
+ *
+ * @example {@lang xml}
+ * <pf-tooltip animation="fade" target="#btn-left" placement="left" delay="100" duration="150" container="#container"></pf-alert>
```

Comment:
I like having container as an attribute for the added flexibility there, but am fine with changing the default container to the parent element (instead of body). Was just following the BootstrapNative pattern with that...